fastlane/Fastfile

```diff
@@ -144,6 +144,7 @@ platform :ios do
         workspace: "./ios/NewExpensify.xcworkspace",
         skip_profile_detection: true,
         scheme: "NewExpensify",
+        xcargs: { :PROVISIONING_PROFILE_SPECIFIER => "chat_expensify_adhoc", },
```

Comment:
It seems like we are already exporting the provisioning profile below. What makes us think that exporting it here will fix the issue?

Reply:
TIL